### PR TITLE
Added DateTimeOffsetConverter.md to the Docs...

### PR DESCRIPTION
**Pull request by CliffAgius**
Documentation added for .NET MAUI Community Toolkit proposals:

#https://github.com/CommunityToolkit/Maui/issues/56

### CODE REVIEW

**Comment by PRMerger12** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 File Change Percent | This PR contains file(s) with more than 20% file change.

**Comment by CliffAgius** [comment]
Hey @bijington here is my first docs page sorry last week became very hectic so only just got around to writing it.  Let me know if it's ok or not.
Thanks

**Comment by jfversluis** [comment]
I think we also need to add this to the TOC?

**Comment by CliffAgius** [comment]
Didn't spot that @jfversluis but added now I think I have all the links in there now but let me know.

**Comment by PRMerger14** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

File: docs/maui/converters/datetimeoffsetconverter.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+             xmlns:toolkit="http://schemas.microsoft.com/dotnet/2022/maui/toolkit"
+             x:Class="MyLittleApp.MainPage">
+    <ContentPage.Resources>
+         <ResourceDictionary>
+             <toolkit:DateTimeOffsetConverter x:Key="DateTimeOffsetConverter" />
+         </ResourceDictionary>
+    </ContentPage.Resources>
+
+    <VerticalStackLayout>
+            <Label Text="The DatePicker bellow is bound to a Property of type DateTimeOffset."
```
Comment (jfversluis) [inline docs/maui/converters/datetimeoffsetconverter.md:32]:
```suggestion
            <Label Text="The DatePicker below is bound to a Property of type DateTimeOffset."
```

**Review by jfversluis — APPROVED** [review]
All good, thank you so much Clifford!

**Comment by PRMerger4** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Pull request merged**